contracts/Party.sol

```diff
@@ -79,23 +79,24 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
 
     // ============ Public Not-Mutated Storage ============
 
-    // market wrapper contract exposing interface for
-    // market auctioning the NFT
-    IMarketWrapper public marketWrapper;
     // NFT contract
     IERC721Metadata public nftContract;
-    // Fractionalized NFT vault responsible for post-auction value capture
-    ITokenVault public tokenVault;
-    // ID of auction within market contract
-    uint256 public auctionId;
     // ID of token within NFT contract
     uint256 public tokenId;
+    // Fractionalized NFT vault responsible for post-purchase experience
+    ITokenVault public tokenVault;
     // the address that will receive a portion of the tokens
-    // if the PartyBid wins the auction
+    // if the Party successfully buys the token
     address public splitRecipient;
```

Comment:
would you be open to adding a helper function that returns all of public variables? would be helpful for the frontend (altho can still do it using batch rpc calls, less convenient)